docs/sdks/cli/GETTING_STARTED.md

```diff
@@ -98,7 +98,7 @@ $ appwrite users list
 
 To create a document you can use the following command 
 ```sh
-$ appwrite database createDocument --collectionId <ID> --documentId 'unique()' --data '{ "Name": "Iron Man" }' --permissions 'read("any")' 'read("team:abc")'
+$ appwrite databases create-document --database-id=<DATABASE_ID> --collection-id=<COLLECTION_ID> --document-id="unique()" --data '{"name": "Walter O Brein"}'
```

Comment:
Let's keep the permissions example too for an example on how to pass array values

Reply:
Re-added the perms